var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -860,6 +1025,44 @@ def configure_args(self):
 
         return options
 
+
+    def my_flag_handler(self, name, flags):
+        if self.spec.satisfies("+stage1"):
+            if name == "cxxflags":
+                cxx_inc_base = self.spec.prefix.include.join("c++").join(str(self.spec.version))
+                flags.append('-I')
+                flags.append(cxx_inc_base)
+                flags.append('-I')
+                flags.append(join_path(cxx_inc_base, self.spec.target_triple))
+                flags.append('-I')
+                flags.append(join_path(cxx_inc_base, "backward"))
+            elif name == "ldflags":
+                flags.extend([
+                    '-L' + self.spec.prefix.lib,
+                    '-L' + self.spec.prefix.lib64,
+                    '-Wl,-rpath,' + self.spec.prefix.lib,
+                    '-Wl,-rpath,' + self.spec.prefix.lib64,
+                ])
+        return (flags, None, None)
+
+    flag_handler = my_flag_handler
+    @run_before("configure", when="+stage1")
```

Comment:
I still don't fully understand why this is done. Can you leave a comment?

Reply:
I could swap it to make the method explicit. LSP doesn’t like that we replace a variable with a method, it’s an unsafe override.